packages/web3.js-plugin-fee-estimation/src/plugin.spec.ts

```diff
@@ -0,0 +1,241 @@
+import { beforeAll, describe, expect, test } from 'vitest'
+import Web3, { Contract, FMT_BYTES, FMT_NUMBER } from 'web3'
+import {
+  l2StandardBridgeABI,
+  l2StandardBridgeAddress,
+  optimistABI,
+  optimistAddress,
+} from '@eth-optimism/contracts-ts'
+
+import OptimismFeeEstimationPlugin from './plugin'
```

Comment:
```suggestion
import {OptimismFeeEstimationPlugin} from './plugin'
```
Let's not import it as a default import so later if we add more plugins to this package we don't need to make a breaking change